pom.xml

```diff
@@ -45,10 +45,10 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
     <maven-project-info-reports-plugin.version>2.9</maven-project-info-reports-plugin.version>
-    <jenkins.version>1.625.3</jenkins.version>
-    <java.level>7</java.level>
-    <jgit.version>4.5.4.201711221230-r</jgit.version>
-    <jgit.javadoc.version>4.5.0.201609210915-r</jgit.javadoc.version>
+    <jenkins.version>2.60.1</jenkins.version>
```

Comment:
I tend to recommend 2.60.3. No one should be running .1/.2 and people running .3 are better served by a plugin which is actually tested against the fixes they have.

Reply:
done